stable/mongodb-replicaset/README.md

```diff
@@ -163,14 +163,14 @@ messy-hydra-mongodb-0 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.def
 
 Kill the primary and watch as a new master getting elected.
 ```console
-$ kubectl delete pod $RELEASE_NAME-mongodb-0
+$ kubectl delete pod $RELEASE_NAME-mongodb-replicaset-0
 
 pod "messy-hydra-mongodb-0" deleted
 ```
 
 Delete all pods and let the statefulset controller bring it up.
 ```console
-$ kubectl delete po -l app=mongodb
+$ kubectl delete po -l app=mongodb-replicaset
```

Comment:
It may work in many cases but not in all. The `app` label may not fully qualify the pods. As soon as you deploy the chart more than once in the same namespace, you'll be in trouble.